UPGRADE-8.0.md

```diff
@@ -503,7 +503,7 @@ Translation
    -$loader->setCsvControl(';', '"', '\\');
    +$loader->setCsvControl(';', '"');
    ```
-
+ * Remove `TranslatableMessage::__toString()` method, use `trans()` instead
```

Comment:
Actually, the direct replacement for `TranslatableMessage::__toString()` is `TranslatableMessage::getMessage()`, when the goal is to get the translation key (rather than the translated message)

Reply:
updated to suggest both thanks